scripts/compare_gas.py

```
@@ -0,0 +1,69 @@
+import re
+import sys
+
+TAIKO_GAS_REPORT = "gas-reports/test_inbox_measure_gas_used.txt"
+MINIMAL_GAS_REPORT = "gas-reports/taiko_inbox_publish.txt"
+
+def extract_gas_value(file_path, pattern):
+    try:
+        with open(file_path, 'r') as f:
+            content = f.read()
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not found: {file_path}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)